### PR TITLE
Add Page Builder migration message banner

### PR DESCRIPTION
**Pull request by bdenham**
Added banner message to PB docs about migration tool not being available until a later date.

## This PR is a:

- [x] Content update

## Summary

When this pull request is merged, it will add a banner to the Page Builder docs informing developers that the migration tool will be released at a later date as well as `page-builder-migration` to the ignore file.

### CODE REVIEW

**Comment by bdenham** [comment]
Because we are not yet ready to add the migration docs to the production devdocs master branch. However, Dave did want to post the message about migration to the public will we finish up.

**Comment by bdenham** [comment]
@jeff-matthews: Sorry for the confusion. I retitled this PR to better reflect the changes. I will remove the same .gitignore change from the `migration` branch.

File: _includes/layout/after-site-header.html [hunk 1]
```diff
@@ -3,3 +3,9 @@
   <a href="http://devdocs.magento.com/guides/v2.0/release-notes/ReleaseNotes2.0.18CE.html" target="_blank">Magento 2.0.18</a> is the final 2.0.x release. After March 2018, Magento 2.0.x will no longer receive security patches, quality fixes, or documentation updates.</br> To maintain your site's performance, security, and PCI compliance, upgrade to the latest version of Magento.</a>
 </div>
 {% endif %}
+
+{% if page.group == "page-builder" %}
+<div class="message-banner">
+  Please note: The <strong>BlueFoot content to Page Builder data migration tool<strong> is not yet released. Stay tuned for more information.
```
Comment (jeff-matthews) [inline _includes/layout/after-site-header.html]: The phrasing is a little awkward. Consider rewriting:

> Documentation about migrating data from Bluefoot to Page Builder is not yet available. Stay tuned for more information.

Comment (bdenham) [reply _includes/layout/after-site-header.html]: Yeah, it's more than just the documentation that isn't available, it's the module/tool itself. How about:
> The module for migrating BlueFoot content to Page Builder is not yet available. Stay tuned for more information.

**Comment by bdenham** [comment]
running tests

File: _includes/layout/after-site-header.html (hunk 1 above)
Comment (jeff-matthews) [reply _includes/layout/after-site-header.html]: Sounds good!

**Pull request merged**